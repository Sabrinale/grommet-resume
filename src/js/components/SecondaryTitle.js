import React from 'react'

import { Heading } from 'grommet'

export const SecondaryTitle = ({ children }) => (
  <Heading level={4}>{children}</Heading>
)
