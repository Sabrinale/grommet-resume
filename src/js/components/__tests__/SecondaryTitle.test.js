import React from 'react'
import renderer from 'react-test-renderer'

import { SecondaryTitle } from '../SecondaryTitle'

test('renders secondary title', () => {
  const component = renderer.create(<SecondaryTitle>test</SecondaryTitle>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
