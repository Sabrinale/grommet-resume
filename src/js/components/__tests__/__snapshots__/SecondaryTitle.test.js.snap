// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders secondary title 1`] = `
<h4
  className="StyledHeading-sc-1rdh4aw-0 hlZfma"
>
  test
</h4>
`;
